src/constants.ts

```diff
@@ -0,0 +1 @@
+export const ADMIN_ROLE_ID = '384ce83c-adce-446a-bde2-c2473c30a453';
```

Comment:
It would be better to get this ID from the options. 
Plus, we should set the id when creating the entry in the db.